containers/pixi/README.md

```diff
@@ -0,0 +1,89 @@
+> They do not exist in the built container image.
+
+## Workspace creation
+
+The Pixi workspace in this example was created with the equivalent of the following Pixi commands (run on a `linux-64` platform machine with an NVIDIA GPU and driver present (providing the [`__cuda` virtual package](https://docs.conda.io/projects/conda/en/stable/user-guide/tasks/manage-virtual.html)))
+
+```bash
+pixi init
+pixi workspace system-requirements add cuda 12.9
+pixi add pytorch-gpu torchvision 'cuda-version 12.9.*'
+pixi task add --description "Train a PyTorch CNN classifier on the MNIST dataset" train "python ./main.py --epochs 20 --save-model"
+```
+
+> [!IMPORTANT]
+> If you are on a platform that does support CUDA (e.g., Linux or Windows), but do not have a NVIDIA GPU and driver present on the host machine, then you will need to provide the `CONDA_OVERRIDE_CUDA` override environment variable for the `__cuda` virtual package  to resolve the dependency requirements.
+>
+> ```bash
+> pixi init
+> pixi workspace system-requirements add cuda 12.9
+> CONDA_OVERRIDE_CUDA=12.9 pixi add pytorch-gpu torchvision 'cuda-version 12.9.*'
+> pixi task add --description "Train a PyTorch CNN classifier on the MNIST dataset" train "python ./main.py --epochs 20 --save-model"
+> ```
+
+> [!NOTE]
+> If you were running these commands from an `osx-arm64` or `win-64` platform you would need to specify that `linux-64` is the target platform
+>
+> ```bash
+> pixi init
+> pixi workspace platform add linux-64
+> pixi workspace system-requirements add cuda 12.9
+> pixi add --platform linux-64 pytorch-gpu torchvision 'cuda-version 12.9.*'
+> pixi task add --description "Train a PyTorch CNN classifier on the MNIST dataset" train "python ./main.py --epochs 20 --save-model"
+> ```
+
+## Docker
+
+### Build Docker container image
+
+Build a Docker container image from the `Dockerfile` that installs the Pixi environment `ENVIRONMENT` and CUDA version `CONDA_OVERRIDE_CUDA`.
+The default values are those set in the `Dockerfile`.
+
+```
+docker build --file Dockerfile --platform linux/amd64 --tag ghcr.io/<your github org>/templates-gpu:mnist-gpu-noble-cuda-12.9 .
+```
+
+and then [authenticate](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#authenticating-to-the-container-registry) and publish your built container image to your GitHub container registry
+
+```
+docker push ghcr.io/<your github org>/templates-gpu:mnist-gpu-noble-cuda-12.9
+```
+
+Note that this manual build and publish process is slow.
+It is recommended to have this step be done through a [continuous integration and continuous delivery (CI/CD) workflow](https://carpentries-incubator.github.io/reproducible-ml-workflows/pixi-deployment.html#automation-with-github-actions-workflows).
```

Comment:
Link out to https://carpentries-incubator.github.io/reproducible-ml-workflows/pixi-deployment.html#automation-with-github-actions-workflows to provide more context.